files/packs.json

```diff
@@ -684,6 +684,24 @@
         "file": "SkyMobs.zip",
         "hash": "f226121d849d529fd2c23c47dd9b552b"
     },
+    {
+        "id": "nosplashtexts",
+        "display": "NoSplashTexts",
+        "creator": "BlockyRick, multiverison port by Erymanthus",
+        "description": "Multiversion resource pack to remove title screen splash text.",
```

Comment:
Why do we care if it's multiversion

Reply:
in the incredible offchance that skyclient shifts versions bc admins decided "yes lets just do the skyblock 1.17 swap"